event-input.h

```diff
@@ -162,6 +162,46 @@ static const int *const touch_event_keys[]= {
 	touch_keys,
 };
 
+static const int keyboard_event_types[] = {
+	EV_KEY,
+	/** No more entries */
+	-1
+};
+
+static const int keyboard_keys[]= {
+	KEY_Q,
+	KEY_W,
+	KEY_E,
+	KEY_R,
+	KEY_T,
+	KEY_Y,
+	KEY_U,
+	KEY_I,
+	KEY_O,
+	KEY_P,
+	KEY_A,
+	KEY_S,
+	KEY_D,
+	KEY_F,
+	KEY_G,
+	KEY_H,
+	KEY_J,
+	KEY_K,
+	KEY_L,
+	KEY_Z,
+	KEY_X,
+	KEY_C,
+	KEY_V,
+	KEY_B,
+	KEY_N,
+	KEY_M,
+	-1
```

Comment:
Another issue I see here - didn't try it, but most-probably we'll end up with keyboard hook attached twice on n900 for the HW kbd - once because of the existing logic and once more - because of that new logic. That shall be fixed in the same commit.

Reply:
No there is no way the code ever matches a device twice, every match is guarded by an else if

Reply:
the old code comes first so it takes precedence

Reply:
ok